docs/JSONDocumentation.md

```diff
@@ -0,0 +1,379 @@
+
+# JSON documentation
+
+This document contains an overview of the possible keys in the intermediary JSON file. It also contains a brief explanation of the meaning of the key in the different programming languages the analyser accepts. Unless indicated otherwise, the terminology used is based on DOTNET.
+
+## Member
```

Comment:
I expected `Type` (from our class `TypeDescription` and their class `TypeDescription`) to be the top-level element, and then each of those having multiple members.

Reply:
The reason I didn't do that is because a header 'Type' is fairly meaningless without additional context. Maybe Type Member? Or Top Level: Type?

Reply:
I think just `Type` is fine; the [guide](https://github.com/eNeRGy164/LivingDocumentation/blob/main/docs/guide.md) from the upstream project also suggests that the JSON is an array of types:

```
var types = JsonConvert.DeserializeObject<List<TypeDescription>>(...);
```